protocol/communities/persistence_test_helpers.go

```diff
@@ -110,7 +113,7 @@ func (p *Persistence) saveRawCommunityRow(rawCommRow RawCommunityRow) error {
 
 func (p *Persistence) saveRawCommunityRowWithoutSyncedAt(rawCommRow RawCommunityRow) error {
 	_, err := p.db.Exec(
-		`INSERT INTO communities_communities ("id", "private_key", "description", "joined", "verified", "muted") VALUES (?, ?, ?, ?, ?, ?)`,
+		`INSERT INTO communities_communities ("id", "private_key", "description", "joined", "verified", "muted", "last_opened_at") VALUES (?, ?, ?, ?, ?, ?, ?)`,
```

Comment:
The `rawCommRow` struct should have its `LastOpenedAt` property set. Otherwise you won't actually do anything with the value you've pulled from the DB.

Reply:
Done